crud/helpers.py

```diff
@@ -45,3 +54,38 @@ async def wrapper(*args, db, **kwargs):
             await db.close()
 
     return wrapper
+
+
+def send_mail(to_address, subject, text):
+    """
+    Send emails using the Mailtrap service using the API token configured
+    in `mail_token`.
+    """
+    mail = mt.Mail(
+        sender=mt.Address(email=mail_from_address, name="Todo-app"),
+        to=[mt.Address(email=to_address)],
+        subject=subject,
+        text=text
+    )
+    client = mt.MailtrapClient(token=mail_token)
+
+    client.send(mail)
+
+
+def raise_helper(status_code, element=None, element_property=None):
```

Comment:
I would advise against using this function. It makes the code harder to read, as we have to jump to this function to check what error will be raised.

I understand why you did this—you’re trying to remove redundant code—but I think code readability is more important. However, this is a minor issue, so if you choose to leave it as is, I won’t insist.